npm/pkg/controlplane/controllers/v1/controllers_v1_test.go

```diff
@@ -8,13 +8,14 @@ import (
 	"github.com/Azure/azure-container-networking/npm/ipsm"
 	"github.com/Azure/azure-container-networking/npm/iptm"
 	"github.com/Azure/azure-container-networking/npm/metrics"
+	"github.com/Azure/azure-container-networking/npm/util"
 	"k8s.io/utils/exec"
 )
 
 func TestMain(m *testing.M) {
 	metrics.InitializeAll()
 	realexec := exec.New()
-	iptMgr := iptm.NewIptablesManager(realexec, iptm.NewFakeIptOperationShim())
+	iptMgr := iptm.NewIptablesManager(realexec, iptm.NewFakeIptOperationShim(), util.PlaceAzureChainAfterKubeServices)
```

Comment:
instead of `util.PlaceAzureChainAfterKubeServices` use  `npmconfig.DefaultConfig.Toggles.ShouldPlaceAzureChainFirst`